src/OneExplorer/OneExplorer.ts

```diff
@@ -423,7 +438,7 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
   private fileWatcher = vscode.workspace.createFileSystemWatcher(`**/*`);
 
   private tree: DirectoryNode|undefined;
-  public didHideExtra: boolean = false;
+  public static didHideExtra: boolean = false;
```

Comment:
optional) I am not sure whether making this to `static` of `OneTreeDataProvider` is correct approach. To me this var in `OneTreeDataProvider` is not class var but object var.. and it seems like a quick fix to use factory class. :-)
Some thought is... I think we eventually need some object that stores persistent state of ONE vscode and that is loaded (from persistent storage) when a ONE user reloads vscode. `didHideExtra` could be possibly one in the object (maybe not in `OneTreeDataProvider`).  :-)

Reply:
Yes, I agree with that. Until we introduce some kind of extension-wise context manager, we may face the similar cases. :-D Likewise, I set 'one.explorer:didHideExtra' context. Let's talk about that later on.